docs/content/concepts/layer/control/catalog/sp800-53rev5-example.md

```diff
@@ -0,0 +1,340 @@
+
+In OSCAL, the values of properties must be atomic values, such as strings, numbers or dates, without Markdown (for example) or other kinds of formatting. To extend properties to provide for values with more complex object types represents a challenge to be met, because any representation of data at this level must be in a form as widely accessible as possible across the greatest range of consumers (languages, platforms, serialization formats, and information modeling and programming technologies and paradigms). To mitigate this limitation, where simple qualification by assigning name/value pairs is not enough, controls (and other structures) in OSCAL can also be extended using other generic elements such as `<link>` and `<part>`.
+
+## Links
+
+Following any property sequence come any links proper to this control. Consistently with modern practice, only outbound links are encoded explicitly.
+
+Like parameters and properties, `<link>` elements provide for explicit representation of data given implicitly in published sources. Here, this information is represented by the __Related Controls__ and __References__ sections of the control text. In each case, targets of the links form distinct classes of their own -- they are either cross-referenced controls or enhancements (also `<control>` elements in OSCAL), or references to bibliographic entries listed as back matter. These are distinguished by `rel` values permitting each to be resolved and handled correctly.
+
+On links, `href` follows the semantics of `href` as defined for HTML/HTTP by W3C. `rel` is constrained as an enumerated value by OSCAL.
+
+{{<highlight xml "nowrap=false" >}}
+<link rel="reference" href="#27847491-5ce1-4f6a-a1e4-9e483782f0ef"/>
+<link rel="reference" href="#c7ac44e8-10db-4b64-b2b9-9e32ec1efed0"/>
+<link rel="reference" href="#08b07465-dbdc-48d6-8a0b-37279602ac16"/>
+<link rel="reference" href="#cec037f3-8aba-4c97-84b4-4082f9e515d2"/>
+<link rel="reference" href="#4c0ec2ee-a0d6-428a-9043-4504bc3ade6f"/>
+<link rel="reference" href="#7f473f21-fdbf-4a6c-81a1-0ab95919609d"/>
+<link rel="related" href="#ia-1"/>
+<link rel="related" href="#pm-9"/>
+<link rel="related" href="#pm-24"/>
+<link rel="related" href="#ps-8"/>
+<link rel="related" href="#si-12"/>
+{{< / highlight >}}
+
+No rule in OSCAL prevents a link from targeting any resource or target whatsoever; but catalog authors can be expected to manage their links according to regimes appropriate to their catalogs. In the case of SP 800-53, internal links (from one control to another) are represented as __Related Controls__ in the formatted source. These are easily indicated via `href` fragment identifiers - `#ia-1` referencing the entity with `id` value `ia-1`, which of course happens to be control IA-1.
```

Comment:
It is both a constraint, and a rule. Indeed, not only does no constraint in OSCAL prevent users from making links freely, but also, no rule does either -- not even a rule that is not being enforced. (Ponder that!)

Reply:
Maybe I should be more clear: when we implement rules sooner rather than later, this blurs the lines what thing we describe in prose is actually `rule` or `constraint` if you look at OSCAL and/or its Metaschema? This seems specifically a description of "this won't be a constraint."

That being said, this is a minor piece of editorial feedback. I will silence this PR feedback and suggestion.

Reply:
This page is for beginners, pretty much. For anyone deep enough to be confused, we can suggest the term "rule" here is used here in a non-technical sense. Kind of like the term "link", which arguably encompasses both `link` and `a` objects in the model, without explanation. Because there are "rules" in one part of a system does not mean there are not also "rules" in some other part of the system or applicable in some other layer.

OSCAL rules do not forbid you from using links as you wish; this includes permitting you to make rules for them. Interestingly, what you are (implicitly) pointing out is that in fact there is indeed some regularity here in SP800-53 *alone*, perhaps not appropriate to other catalogs, around usage of `link[@rel='related']` and `link[@rel='reference']`. However, those rules for SP800-53 are made within the context of OSCAL rules that (by saying nothing) permit you to assert your own rules.

Reply:
I think we should avoid using "rule", since this will have a different meaning when we support rules and tests in OSCAL. I like "constraints" better.